### PR TITLE
Fix for custom fields containing multiple lines

### PR DESCRIPTION
**Pull request by jerash**
Fix for issue https://github.com/CZ-NIC/python-rt/issues/11
I use that in production for 3 months and no problem so far, hope it can be usefull

### CODE REVIEW

**Comment by e3rd** [comment]
Cool, could you put a line in CHANGES under `v2.1.0, CURRENT_DATE`?

**Comment by jerash** [comment]
i'm sorry idon't understand how to do what you ask

**Comment by e3rd** [comment]
Don't mind, I'll do that. I plan to release a new minor version with this and #60 when ready.

File: rt/rt.py [hunk 1]
```diff
@@ -554,16 +566,27 @@ def get_ticket(self, ticket_id: typing.Union[str, int]) -> typing.Optional[dict]
...
+            lastheader = ''
             for i in range(req_id, len(msg)):
                 if ': ' in msg[i]:
                     header, content = self.split_header(msg[i])
                     pairs[header.strip()] = content.strip()
+                    lastheader = header
+                elif len(msg[i]) == 0:
+                    # print("Empty Line")
+                    pass
+                elif msg[i] == "Cc:" or msg[i] == "AdminCc:":
```
Comment (e3rd) [inline rt/rt.py:578]: When does `msg[i] == "Cc:"` happen? It seems to me the line 571 picks every `msg` that has a colon. Is it true that "Cc:" cannot be multiline?

File: rt/rt.py [hunk 2]
```diff
@@ -554,16 +566,27 @@ def get_ticket(self, ticket_id: typing.Union[str, int]) -> typing.Optional[dict]
...
+                    # print("Ignore")
+                    pass
+                else:
+                    # print("CANNOT PAIR with -" + msg[i] + "- using last header " + lastheader)
```
Comment (e3rd) [inline rt/rt.py:582]: Join else-if to elif and get rid of "print" (or reformulate it to be a comment), not a leftover.

**Review by e3rd — CHANGES_REQUESTED** [review]
The PR you've submitted concerns an important task. Would you be able to add a test case into `test_rt.py` for it please?

**Comment by e3rd** [comment]
As this PR is stalled, it was superseded by #64

**Pull request closed without merging**